radicle-httpd/src/api/v1/delegates.rs

```diff
@@ -48,10 +48,10 @@ async fn delegates_projects_handler(
                 return None;
             }
 
-            let Ok(issues) = Issues::open(ctx.profile.public_key, &repo) else { return None };
-            let Ok(issues) = (*issues).count() else { return None };
-            let Ok(patches) = Patches::open(ctx.profile.public_key, &repo) else { return None };
-            let Ok(patches) = (*patches).count() else { return None };
+            let Ok(issues) = Issues::open(ctx.profile.public_key, &repo) else { return None; };
+            let Ok(issues) = issues.group_by_state() else { return None };
+            let Ok(patches) = Patches::open(ctx.profile.public_key, &repo) else { return None; };
+            let Ok(patches) = patches.group_by_state() else { return None; };
```

Comment:
why first `return None` doesn't have `;`? tbh I prefer without for all cases. which also applies to changes in [radicle-httpd/src/api/v1/projects.rs](https://github.com/radicle-dev/heartwood/blob/eebb459f4990440471783f031982e517ef04a65d/radicle-httpd/src/api/v1/projects.rs).

Reply:
Hey @xphoniex nice catch, yeah I think I'll remove the redundant semicolons

Reply:
20773cf solves this